Jenkinsfile_CNP

```diff
@@ -17,7 +17,12 @@ def secrets = [
                 secret('OAUTH2-CLIENT-SECRET', 'OAUTH2_CLIENT_SECRET'),
                 secret('OAUTH2-CLIENT-AUTH', 'OAUTH2_CLIENT_AUTH'),
                 secret('OAUTH2-CLIENT-ID', 'OAUTH2_CLIENT_ID'),
-                secret('LD-SDK-KEY', 'LD_SDK_KEY')
+                secret('LD-SDK-KEY', 'LD_SDK_KEY'),
+                secret('professional-api-POSTGRES-HOST', 'FUNC_DATABASE_HOST'),
+                secret('professional-api-POSTGRES-PASS', 'FUNC_DATABASE_PASS'),
+                secret('professional-api-POSTGRES-USER', 'FUNC_DATABASE_USER'),
+                secret('professional-api-POSTGRES-DATABASE', 'FUNC_DATABASE_NAME'),
+                secret('professional-api-POSTGRES-PORT', 'FUNC_DATABASE_PORT')
```

Comment:
add the entries in the jenkins nighly file also

Reply:
Earlier it was decided to run script from only master/aat build, now it will run only in nightly.